internal/elasticsearch/ingest/processor_reroute_data_source.go

```diff
@@ -0,0 +1,127 @@
+package ingest
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/models"
+	"github.com/elastic/terraform-provider-elasticstack/internal/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func DataSourceProcessorReroute() *schema.Resource {
+	processorSchema := map[string]*schema.Schema{
+		"id": {
+			Description: "Internal identifier of the resource.",
+			Type:        schema.TypeString,
+			Computed:    true,
+		},
+		"field": {
```

Comment:
The reroute resource doesn't have a `field` attribute. Add the `destination`, `dataset`, and `namespace` attributes as included in the docs (https://www.elastic.co/docs/reference/enrich-processor/reroute-processor)